Client/Ecosia/UI/NTP/NTPTooltip.Highlight.swift

```diff
@@ -34,10 +34,10 @@ extension NTPTooltip {
 
     }
 
-    class func highlight(for user: Core.User = User.shared,
-                         isInPromoTest: Bool = DefaultBrowserExperiment.isInPromoTest()) -> NTPTooltip.Highlight? {
+    class func highlight(for user: Core.User = User.shared) -> NTPTooltip.Highlight? {
+        
         // on first start, when we show the default browser promo, no highlight should be shown
```

Comment:
I guess this comment can be removed

Reply:
@lucaschifino you mean because it's self-explanatory given the line below?

Reply:
I mean because `on first start, when we show the default browser promo ...` is no longer valid since it's no longer shown on first start. That's why the logic on the line below was even removed, isn't it?

Reply:
Maybe just removing `when we show the default browser promo` and leaving 
```suggestion
        // on first start no highlight should be shown
```

Reply:
Correct 👍 . Only the "is first time" check is left 👍 . Will remove the comment